hbase-common/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfo.java

```diff
@@ -38,6 +38,9 @@
   // Keep servers in a sorted set so has an expected ordering when displayed.
   private final SortedSet<Address> servers;
   // Keep tables sorted too.
+
+  // TODO: Don't understand why all these should be deprecated. we have table -> rsgroup mapping.
```

Comment:
Because we do not store the table informations in RSGroupInfo anymore, if you use the new API to get RSGroupInfo, the tables will always be empty.